docs/source/pinning.rst

```diff
@@ -0,0 +1,65 @@
+===============
+Release Pinning
+===============
+
+Firefox update supports a feature known as update pinning.
+The purpose of this feature is to allow users (primarily enterprise users) to set a version beyond which installations should not update.
+Two types of pins can be set: major version pins (ex: ``102.``) and minor version pins (ex: ``102.1.``).
+
+This feature is a bit more complicated than simply not installing an update if it is beyond the specified version.
+The problem with that approach is that, without the changes made to accommodate this feature, Balrog will always provide the newest version that the installation can be updated to.
+Balrog needs to be aware of what version the installation is pinned to.
+To understand why, consider an installation at version 102 that is pinned to version 103 when the newest version is 104.
+If Balrog wasn't aware of the pinned version, the installation would ask for updates and would get the update to 104 in response.
+Since that is newer than the pin, instead of updating to 103, it wouldn't update at all.
+To accommodate update pinning, the application needs to send the pin as part of the update URL so that Balrog knows the correct version to return in cases like this.
+
+-----------------
+How Pinning Works
+-----------------
+
+~~~~~~~~~~~~~~~~~~
+Release Automation
+~~~~~~~~~~~~~~~~~~
+
+The process starts when a new release is built.
+When the build has completed, Release Automation automatically submits the release to Balrog.
+The Balrog submission Taskcluster jobs specify one or more channels that the release should be pinned for (See `Bug 1762979 <https://bugzilla.mozilla.org/show_bug.cgi?id=1762979>`_ for details).
```

Comment:
I find "The Balrog submission Taskcluster jobs" a little awkward, but I'm not sure how to say it better.

Reply:
I reworded it as "The Balrog submission jobs (in Taskcluster)". Maybe that's a little bit better?

Reply:
wfm. Thanks.